### DISCUSSION TITLE
Unet models and clip directory can not seen by Forge

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/stable-diffusion-webui-forge#1451] [Q&A] [answered]
My Unet models stored in C:\SD_AI\ComfyUI\ComfyUI\models\unet
and Clip stored in C:\SD_AI\ComfyUI\ComfyUI\models\clip
Forge can not see them 
I have managed to direct ComfyUI checkpoints and loras to Forge
set COMMANDLINE_ARGS= --theme dark --ckpt-dir "C:/SD_AI/ComfyUI/ComfyUI/models/checkpoints" --lora-dir "C:\SD_AI\ComfyUI\ComfyUI\models\loras" --models-dir "C:/SD_AI/ComfyUI/ComfyUI/models" --vae-path "C:/SD_AI/ComfyUI/ComfyUI/models/vae"

I there a way to direct unet models and clips to Forge
I have tried many ARG but none works

**Comment by DenOfEquity** [comment, marked as answer]
Does `--vae-path` work for this? I thought it should be `--vae-dir`.
As I understand it, Forge treats unets same as checkpoints, expects them in same directory. So, there's no command line argument to change this.
Command line argument to set location for text encoders now added: `--text-encoder-dir`.

**Reply by wisdombox44** [reply]
Thank you, VAE and text encoders worked with the arguments provided. However, I am still looking for a solution for the UNet models stored in the ComfyUI models folder. They only work if I move the UNet models to the ComfyUI checkpoints folder, but then the ComfyUI workflows with the UNet models don't work. Is there an argument for specifying the UNet models?
